### PR TITLE
Problem: sdkerrors.Wrap is deprecated

### PR DESCRIPTION
**Pull request by mmsqe**
Solution:
- replace with `cosmossdk.io/errors`

👮🏻👮🏻👮🏻 !!!! REFERENCE THE PROBLEM YOUR ARE SOLVING IN THE PR TITLE AND DESCRIBE YOUR SOLUTION HERE !!!! DO NOT FORGET !!!! 👮🏻👮🏻👮🏻

# PR Checklist:

- [ ] Have you read the [CONTRIBUTING.md](https://github.com/crypto-org-chain/chain-main/blob/master/CONTRIBUTING.md)?
- [ ] Does your PR follow the [C4 patch requirements](https://rfc.zeromq.org/spec:42/C4/#23-patch-requirements)?
- [ ] Have you rebased your work on top of the latest master? 
- [ ] Have you checked your code compiles? (`make`)
- [ ] Have you included tests for any non-trivial functionality?
- [ ] Have you checked your code passes the unit tests? (`make test`)
- [ ] Have you checked your code formatting is correct? (`go fmt`)
- [ ] Have you checked your basic code style is fine? (`golangci-lint run`)
- [ ] If you added any dependencies, have you checked they do not contain any known vulnerabilities? (`go list -json -m all | nancy sleuth`)
- [ ] If your changes affect the client infrastructure, have you run the integration test?
- [ ] If your changes affect public APIs, does your PR follow the [C4 evolution of public contracts](https://rfc.zeromq.org/spec:42/C4/#26-evolution-of-public-contracts)?
- [ ] If your code changes public APIs, have you incremented the crate version numbers and documented your changes in the [CHANGELOG.md](https://github.com/crypto-org-chain/chain-main/blob/master/CHANGELOG.md)?
- [ ] If you are contributing for the first time, please read the agreement in [CONTRIBUTING.md](https://github.com/crypto-org-chain/chain-main/blob/master/CONTRIBUTING.md) now and add a comment to this pull request stating that your PR is in accordance with the [Developer's Certificate of Origin](https://github.com/crypto-org-chain/chain-main/blob/master/CONTRIBUTING.md#developer-certificate-of-originn).

Thank you for your code, it's appreciated! :)

### CODE REVIEW

File: app/ante/ante.go [hunk 1]
```diff
@@ -1,6 +1,7 @@
 package ante
 
 import (
+	errorsmod "cosmossdk.io/errors"
```
Comment (yihuang) [inline app/ante/ante.go:4]: Shall we just use it as `"errors"`, only rename it when there's ambiguity?

Comment (mmsqe) [reply app/ante/ante.go]: Done

File: app/app.go [hunk 2]
```diff
@@ -21,6 +21,7 @@ import (
 	tmos "github.com/tendermint/tendermint/libs/os"
 	dbm "github.com/tendermint/tm-db"
 
+	errorsmod "cosmossdk.io/errors"
```
Comment (yihuang) [inline app/app.go:24]:
```suggestion
	"cosmossdk.io/errors"
```

File: app/test_helpers.go [hunk 3]
```diff
@@ -18,6 +18,7 @@ import (
 	tmtypes "github.com/tendermint/tendermint/types"
 	dbm "github.com/tendermint/tm-db"
 
+	errorsmod "cosmossdk.io/errors"
```
Comment (yihuang) [inline app/test_helpers.go:21]:
```suggestion
	"cosmossdk.io/errors"
```

File: x/cronos/handler.go [hunk 4]
```diff
@@ -3,6 +3,7 @@ package cronos
 import (
 	"fmt"
 
+	errorsmod "cosmossdk.io/errors"
```
Comment (yihuang) [inline x/cronos/handler.go:6]:
```suggestion
	"cosmossdk.io/errors"
```

File: x/cronos/keeper/keeper.go [hunk 5]
```diff
@@ -4,6 +4,7 @@ import (
 	"fmt"
 	"strings"
 
+	errorsmod "cosmossdk.io/errors"
```
Comment (yihuang) [inline x/cronos/keeper/keeper.go:7]:
```suggestion
	"cosmossdk.io/errors"
```

Comment (mmsqe) [reply x/cronos/keeper/keeper.go]: Thanks, should be clean now

**Review by yihuang — APPROVED** [review]
LGTM

**Pull request merged**